service-mesh/app-src/apps/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ -z $1 ]; then
+    docker_repo=$DOCKER_REPO
+fi
```

Comment:
what if an arg were passed?  Why not just something like `docker_repo=${DOCKER_REPO:='ecommerce'}`

Reply:
Changed this to exit if the var isn't set. I think it should be required to set the variable rather than having a default